env.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 from typing import Tuple, List, Dict, Any, Union
 from actions import ACTIONS
+from random import randint
```

Comment:
Let's use np.random instead of the regular Python random module since we've been using numpy.random in everything else.